ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -3922,6 +3922,16 @@ keys:
                       type: str
                       valid_values:
                       - discard
+      event:
```

Comment:
event needs to have a `type` member and then all sub-members under the `keys` member
```
event:
    type: dict
    keys:
        storm_control_discards:
``` 
`storm_control_discards` should be similar as it seems to be a dict. Also as stated in my github comment these schema changes should exist in the relevant schema_fragment file causing them to get build into here from there. I just can't add a comment in that relevant file as it hasn't changed in the PR